packaging/container-deployment/Containerfile.bpfctl

```diff
@@ -0,0 +1,17 @@
+From rustlang/rust:nightly as bpfctl-build
+WORKDIR /usr/src/bpfd
+COPY ./ /usr/src/bpfd
+
+RUN apt-get update && apt-get install -y protobuf-compiler musl-tools
+
+RUN rustup target add x86_64-unknown-linux-musl
+RUN rustup toolchain install --force-non-host nightly-x86_64-unknown-linux-musl
```

Comment:
I don't think this is needed.

Reply:
It's not :) for some reason I needed this on my local and it got carried over.